ddtrace/internal/http.py

```diff
@@ -34,3 +37,31 @@ class HTTPSConnection(BasePathMixin, httplib.HTTPSConnection):
     """
     httplib.HTTPSConnection wrapper to add a base path to requested URLs
     """
+
+
+@cached()
+def get_wsgi_header(header):
```

Comment:
I'm leaning toward having these be semi-public in `trace_utils` rather than internal as they're used exclusively in integrations.

Thoughts?

Reply:
@Kyle-Verhoog Interesting, I like the idea of making these semi-public in `trace_utils`, but I'm running into a circular import error since `trace_utils` imports from `propagation.http`, which also imports `get_wsgi_header`. Do you have any thoughts on how I can avoid this?